grammar.js

```diff
@@ -1796,6 +1831,8 @@ module.exports = grammar({
     private: $ => 'private',
     protected: $ => 'protected',
 
+    macro_expression: $ => seq('{{', choice($.splat, $.double_splat, $._expression), '}}'),
```

Comment:
We need to be really careful about terminology here: if this (`{{ }}`) is a macro expression then what is this `{% %}`?

Reply:
Macro control, going based off the stdlib parser

Reply:
Actually need to think more about this, as `{% %}` are put inside a `MacroExpression`

Reply:
There will be specific `macro_if` / `macro_begin` for those specific nodes, and `macro_expressions` for multi-line macro code inside `{% ... %}`